challenge_opener/src/main/java/com/example/challengeopener/NewActivityInfoList.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.challengeopener
+
+class NewActivityInfoList {
+    fun getList(): List<NewActivityInfo> = listOf(
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        ),
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        ),
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        ),
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        ),
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        ),
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        ),
+        NewActivityInfo(
+            "A new challenge example",
+            "This is a challenge that will be developed soon",
+            R.drawable.ic_launcher_foreground
+        )
+    )
```

Comment:
@js-gonzalez I understand that you are populating your list with dummy data. However, I think you could do it simpler:

```suggestion
    val list: List<NewActivityInfo> by lazy {
        List(7) { // Number of initial elements
            NewActivityInfo(
                "A new challenge example",
                "This is a challenge that will be developed soon",
                R.drawable.ic_launcher_foreground
            )
        }
    }
```

Reply:
Now this is the Kotlin syntax I should have searched, thanks!